Eatery/Controllers/BRB/SettingsTableViewController.swift

```diff
@@ -0,0 +1,201 @@
+
+//
+//  SettingsTableViewController.swift
+//  Eatery
+//
+//  Created by William Ma on 3/2/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import UIKit
+import SafariServices
+
+struct BRBAccountSettings {
+
+    private static let saveLoginInfoKey = "save_login_info"
+
+    static var saveLoginInfo: Bool {
+        get {
+            return UserDefaults.standard.object(forKey: BRBAccountSettings.saveLoginInfoKey) as? Bool ?? true
```

Comment:
Can you do `return UserDefaults.standard.bool(forKey: ...)` If key does not exist, it will return false